docs/UserGuide.md

```diff
@@ -254,8 +255,15 @@ internship in the company’s list of internships.
 - At least one of the optional fields must be provided.
 
 Examples:
-- `edit int 7 i/1 w/2000 r/Java r/Python`
-- `edit int 4 i/4 j/ML Engineer`
+- Executing: `edit int 1 i/2 w/4800 r/Java r/Python` will change:
+
+![EditInternshipBefore](images/EditInternshipBefore.png)
+
+to:
+
+![EditInternshipAfter](images/EditInternshipAfter.png)
+
+- `edit int 3 i/1 j/Frontend Developer`
```

Comment:
I think it could be better to specify a note saying that the details of the application made for this internship will also be updated accordingly.

Reply:
image of the application?